numpydoc/numpydoc.py

```diff
@@ -124,13 +127,17 @@ def mangle_docstrings(app, what, name, obj, options, lines):
         title_re = re.compile(sixu(pattern), re.I | re.S)
         lines[:] = title_re.sub(sixu(''), u_NL.join(lines)).split(u_NL)
     else:
-        doc = get_doc_object(obj, what, u_NL.join(lines), config=cfg,
-                             builder=app.builder)
-        if sys.version_info[0] >= 3:
-            doc = str(doc)
-        else:
-            doc = unicode(doc)
-        lines[:] = doc.split(u_NL)
+        try:
+            doc = get_doc_object(obj, what, u_NL.join(lines), config=cfg,
+                                 builder=app.builder)
+            if sys.version_info[0] >= 3:
+                doc = str(doc)
+            else:
+                doc = unicode(doc)
+            lines[:] = doc.split(u_NL)
+        except:
+            logger.error('While processing numpydoc for %r', name)
```

Comment:
Suggestion: add `[numpydoc] ` at the front of the message? That's also how sphinx extensions like autodoc/autosummary do it